recipe/meta.yaml

```diff
@@ -113,7 +113,7 @@ requirements:
     - pygments >=2.2.0,<3
     - pynac >=0.7.16
     - pyparsing >=2.2.0,<3
-    - python   # [py==27]
```

Comment:
@isuruf Could you explain why this is/was necessary?

Reply:
I think it was a mistake.